packages/astro/package.json

```diff
@@ -136,7 +136,7 @@
     "strip-ansi": "^7.0.1",
     "supports-esm": "^1.0.0",
     "tsconfig-resolver": "^3.0.1",
-    "vite": "^2.9.14",
+    "vite": "3.0.0",
```

Comment:
Any reason to pin to `3.0.0` exactly? Should we use `~3.0.0`?

Reply:
Ack, didn't catch this! I'm using `^3.0.0` but would we prefer `~3.0.0`?

Reply:
I’m almost sure they’re following semver, but given all the noise around this release I’d be +1 to use ~ until 3.x has had some time to bake. Don’t feel too strongly tho